<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RumoDialog</class>
 <widget class="QDialog" name="RumoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inserir Atributo na Tabela</string>
  </property>
  <property name="styleSheet">
   <string>
    QDialog {
        background-color: #f5f5f5;
    }
    QLineEdit {
        border: 1px solid #cccccc;
        border-radius: 5px;
        padding: 5px;
        background-color: #ffffff;
    }
    QLineEdit:focus {
        border-color: #4a90e2;
    }
    QPushButton {
        background-color: #4a90e2;
        color: white;
        border: none;
        border-radius: 5px;
        padding: 8px;
        min-width: 80px;
    }
    QPushButton:hover {
        background-color: #357abd;
    }
    QTableWidget {
        border: 1px solid #cccccc;
        border-radius: 1px;
        background-color: #ffffff;
        alternate-background-color: #f0f0f0;
    }
    QTableWidget::item {
        padding: 1px;
    }
    QPushButton#salvarButton {
        background-color: #2ecc71;
    }
    QPushButton#salvarButton:hover {
        background-color: #27ae60;
    }
   </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="select_camada">
         <property name="styleSheet">
          <string>color: #333333; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Selecione a Camada:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="coluna_combo"/>
       </item>
       <item>
        <widget class="QComboBox" name="formatacao_combo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="salvarButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Salvar o tra√ßado no mapa</string>
         </property>
         <property name="text">
          <string>Salvar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/plugins/icons/save.png</normaloff>:/plugins/icons/save.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>157</red>
               <green>157</green>
               <blue>157</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>157</red>
               <green>157</green>
               <blue>157</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="icon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
